src/client/Client.cc

```diff
@@ -12215,8 +12262,27 @@ int Client::_getxattr(Inode *in, const char *name, void *value, size_t size,
 		      const UserPerm& perms)
 {
   int r;
+  const VXattr *vxattr = nullptr;
 
-  const VXattr *vxattr = _match_vxattr(in, name);
+  auto is_entire_cluster_quincy = [&]() -> bool {
```

Comment:
This is required to avoid MDS assertions? Please document as a comment. Maybe we can remove someday.

Reply:
ack; comment added in code for the quincy test